drift/info_parser.py

```diff
@@ -218,13 +218,20 @@ def _create_comparison(systems, info_name, reference_id, system_count):
     elif len(system_values) == 1:
         info_comparison = COMPARISON_SAME
 
+    baseline_count = 0
+    for system_id_value in sorted_system_id_values:
+        if system_id_value["is_baseline"]:
+            baseline_count += 1
+
     # override comparison logic for certain fact names
-    if _is_unique_rec_name(info_name) and system_count > 1:
+    if _is_unique_rec_name(info_name) and (system_count - baseline_count) > 1:
```

Comment:
can you add a comment that explains what this is checking for? if i came across this code later I wouldn't remember why it was doing this